### DISCUSSION TITLE
Spot welding

### DISCUSSION THREAD

**Discussion by andrey-pr** [Ralim/IronOS#1938] [General]
Hello. Just a theoretical question: what are you think about adding support of smart spot welding machines like DH30? (its only a theoretical question, actually I even didn't any research if it physically possible at all)

**Comment by Ralim** [comment]
Honestly have never even looked at them 😓 
I'm not against looking into making an alternative firmware for them; but if I did it would likely be its own project as the software isn't going to have a huge amount in common.

Do you have one of these?
If you do, can you see if you get can update the firmware without opening the case (i.e. does it show up on usb)?
